### DISCUSSION TITLE
카카오 인턴 테크 코테 후기

### DISCUSSION THREAD

**Discussion by BanApp** [ghdcksgml1/Algorithm_Study#233] [자유게시판]
1. 카카오 코테는 문제가 넘모 길다.. 다만 문제가 길다고 대충 급하게 읽으면 엄한곳에서 길을 잃고 해맬 가능성이 높기때문에 20분 정도는 1번 부터 5번까지 천천히 정독하는것을 추천한다.

2. 문제가 이런 느낌이다. "너 이 알고리즘 알아?" 이게 아니라 "이 문제 이렇게도 풀 수 있고 저렇게도 풀 수 있는데 둘중 하나는 시간초과남ㅋㅋ" 이거나 "너 @@알고리즘 알지? 우리가 한번 이렇게 만들어봤어 해봐ㅋㅋ" 이런 느낌이다.

3. 카카오에 붙고 싶다면 DP와 그래프를 열심히 하는것이 중요할것 같다.. 그리고 1번이 제일 쉬운건 맞고 5번이 제일 어려운건 맞지만 2,3,4번은 꼭 난이도 순이 아니다. 그래서 문제를 먼저 읽어 보는게 중요한것 같다. 다익스트라 미리 공부좀 해놓을것을 후회중이다..

4. 체감상 백준 골드 하위면 2솔이나 2.5솔 가능할것 같고, 골드 상위권이나 플레5 정도면 무난하게 인턴 코테는 합격할 수준인것 같다.. 본인은 수준미달^_^

5. 그래도 재밌기도 하고 코테를 많이 봐보는게 도움이 많이 될것같으니깐 가을에 우테코 같이 신청하쉴?

**Comment by ghdcksgml1** [comment]
고생하셨습니다~~~~~

**Comment by jusung-c** [comment]
ㅋㅋㅋㅋ후기 리얼하네요
수고했슴다~~
